arm/Microsoft.NetApp/netAppAccounts/capacityPools/readme.md

```diff
@@ -92,6 +92,6 @@ Create a role assignment for the given resource. If you want to assign a service
 
 ## Template references
 
+- [Define resources with Bicep and ARM templates](https://docs.microsoft.com/en-us/azure/templates)
```

Comment:
Where does this come from?

Reply:
Strnage. Doesn't show up for me (locally)